realm/realm-library/src/main/cpp/io_realm_internal_SharedRealm.cpp

```diff
@@ -1,14 +1,14 @@
-#include <object-store/src/sync_manager.hpp>
+#ifdef REALM_SYNC
```

Comment:
Move this block to line 6 and use `""` instead of `<>` to make it look better :P